<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="LAPTOP-5F236986" name="com.cyrano.testcases.Login_valid_credential" tests="3" failures="0" timestamp="21 Jan 2019 09:15:40 GMT" time="27.653" errors="0">
  <testcase name="login_username_check" time="9.503" classname="com.cyrano.testcases.Login_valid_credential"/>
  <testcase name="login_Error_validation" time="10.016" classname="com.cyrano.testcases.Login_valid_credential"/>
  <testcase name="login_Valid_credential_check" time="8.134" classname="com.cyrano.testcases.Login_valid_credential"/>
</testsuite> <!-- com.cyrano.testcases.Login_valid_credential -->
